python/shipDet_conf.py

```diff
@@ -451,6 +452,15 @@ def configure(run,ship_geo):
  timeDet.SetSizeY(2 * ship_geo.TimeDet.DY)
  detectorList.append(timeDet)
 
+ z_coord = [-3100, -2800, -2600]
+  #z_coord = []
+ for i in range(len(z_coord)):
+  scorplane = ROOT.ScoringPlane("sc_pl_" + str(i), ROOT.kTRUE, ROOT.kFALSE, 116.7, 253.5, 0.1)
+  scorplane.SetVetoPointName("sco" + "_" + str(i))
+  scorplane.SetZposition(z_coord[i])
+  detectorList.append(scorplane)
+
+
```

Comment:
Again, let's leave the scoring plane separate from the changes for the SC magnet, please.